manim/utils/docbuild/autoaliasattr_directive.py

```diff
@@ -49,7 +48,9 @@ def smart_replace(base: str, alias: str, substitution: str) -> str:
     occurrences = []
     len_alias = len(alias)
     len_base = len(base)
-    condition = lambda char: (not char.isalnum()) and char != "_"
+
+    def condition(char: str):
```

Comment:
Let's fully type it :)

```suggestion
    def condition(char: str) -> bool:
```